pyvirtualcam/camera.py

```diff
@@ -8,7 +8,12 @@
 from pyvirtualcam.util import FPSCounter
 
 if platform.system() == 'Windows':
-    from pyvirtualcam import _native_windows
+    from pyvirtualcam import _native_windows as _native
+elif platform.system() == 'Darwin':
+    from pyvirtualcam import _native_macos as _native
+elif platform.system() in ['Linux']:
+    import os
```

Comment:
Let's keep this at the top.

Reply:
Only necessary for Linux though; but will do